index.html

```diff
@@ -0,0 +1,191 @@
+        <script src="node_modules/infusion/src/framework/core/js/FluidDebugging.js"></script>
+        <script src="node_modules/infusion/src/framework/core/js/FluidDocument.js"></script>
+        <script src="node_modules/infusion/src/framework/core/js/FluidIoC.js"></script>
+        <script src="node_modules/infusion/src/framework/core/js/DataBinding.js"></script>
+        <script src="node_modules/infusion/src/framework/core/js/ModelTransformation.js"></script>
+        <script src="node_modules/infusion/src/framework/core/js/ModelTransformationTransforms.js"></script>
+        <script src="node_modules/infusion/src/framework/enhancement/js/ContextAwareness.js"></script>
+        <script src="node_modules/infusion/src/framework/enhancement/js/ProgressiveEnhancement.js"></script>
+        <script src="node_modules/infusion/src/framework/core/js/FluidView.js"></script>
+        <script src="node_modules/infusion/src/framework/core/js/FluidView-browser.js"></script>
+        <script src="node_modules/infusion/src/framework/core/js/NewViewSupport.js"></script>
+        <script src="node_modules/infusion/src/framework/core/js/DataSource.js"></script>
+        <script src="node_modules/infusion/src/framework/core/js/ResourceLoader.js"></script>
+        <script src="node_modules/infusion/src/framework/core/js/ResourceLoader-browser.js"></script>
+        <script src="node_modules/infusion/src/framework/renderer/js/fluidParser.js"></script>
+        <script src="node_modules/infusion/src/components/pager/js/Pager.js"></script>
+        <script src="node_modules/infusion/src/components/tooltip/js/Tooltip.js"></script>
+
+        <script type="text/javascript" src="node_modules/new-renderer-demo/src/js/core/jquery.standalone.renderer.js"></script>
+        <script type="text/javascript" src="node_modules/new-renderer-demo/src/js/core/fastXmlPull.js"></script>
+        <script type="text/javascript" src="node_modules/new-renderer-demo/src/js/core/fluidView-template.js"></script>
+        <script type="text/javascript" src="node_modules/new-renderer-demo/src/js/core/fluidNewRenderer.js"></script>
+        <script type="text/javascript" src="node_modules/new-renderer-demo/src/js/core/fluidNewRendererComponents.js"></script>
+        <script type="text/javascript" src="node_modules/new-renderer-demo/src/js/core/htmlParser.js"></script>
+        <script type="text/javascript" src="node_modules/new-renderer-demo/src/js/core/ResourceRewriter.js"></script>
+        <script type="text/javascript" src="node_modules/new-renderer-demo/src/js/client/ClientRenderer.js"></script>
+
+        <script src="node_modules/leaflet/dist/leaflet.js"></script>
+        <script src="node_modules/papaparse/papaparse.js"></script>
+
+        <script src="src/js/colour.js"></script>
+        <script src="src/js/leafletMap.js"></script>
+        <script src="src/js/CSVResource.js"></script>
+        <script src="src/js/covidMap.js"></script>
+
+        <title>Rhetoric</title>
```

Comment:
The demo title probably should be rename to something like "Accessibility Map for COVID-19 assessment centres". The same comment applies to line 58.